android/src/main/java/com/urbanairship/reactnative/EventEmitter.java

```diff
@@ -110,6 +114,48 @@ public void onReactContextInitialized(ReactContext reactContext) {
         }
     }
 
+    /**
+     * Called when a new listener is added for a specified event name.
+     *
+     * @param eventName The event name.
+     */
+    void addAndroidListener(ReactContext reactContext, String eventName) {
+        synchronized (knownListeners) {
```

Comment:
```
void addAndroidListener(ReactContext reactContext, String eventName) {
    synchronized (knownListeners) {
        for (Event event : new ArrayList<>(pendingEvents)) {
            if (event.equals(eventName)) {
                sendEvent(reactContext, event);
                pendingEvents.remove(event);
            }
        }
        listenerCount++;
        knownListeners.add(eventName);
    }
}
```

Reply:
fixed this, it just didn't change the line you commented on